server/index.js

```diff
@@ -1,21 +1,59 @@
-const express = require('express');
-const mongoose = require('mongoose');
-const cors = require('cors');
+const express = require("express");
+const mongoose = require("mongoose");
+const cors = require("cors");
+const Pet = require("./models/Pet");
 
 const app = express();
 const port = process.env.PORT || 5000;
 
 app.use(express.json());
 app.use(cors());
 
-mongoose.connect(process.env.MONGODB_URI || 'mongodb://127.0.0.1:27017/zoonadiya')
-    .then(() => {
-        console.log('Connected to MongoDB');
-    })
-    .catch((error) => {
-        console.error('Failed to connect to MongoDB:', error);
+mongoose
+  .connect(process.env.MONGODB_URI || "mongodb://127.0.0.1:27017/zoonadiya")
+  .then(() => {
+    console.log("Connected to MongoDB");
+  })
+  .catch((error) => {
+    console.error("Failed to connect to MongoDB:", error);
+  });
+
+app.get("/getFilteredPets", (req, res) => {
```

Comment:
this code is dublicated there https://github.com/Osttik/zoohope/pull/27 and shoud be deleted after this https://github.com/Osttik/zoohope/pull/27 pull request